service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -26,8 +48,53 @@ void shouldPrepopulateApplicantDataWhenNoApplicantExists() {
             .data(Map.of("data", "some data"))
             .build();
 
+        given(authTokenGenerator.generate()).willReturn(serviceAuthToken);
+        given(organisationApi.findOrganisationById(userAuthToken, serviceAuthToken))
+            .willReturn(Organisation.builder().build());
+
         AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseDetails);
 
         assertThat(callbackResponse.getData()).containsKey("applicants");
     }
+
+    @BeforeEach
+    void setup() {
+        Organisation organisation = buildOrganisation();
```

Comment:
I think I commented before, you could set organisation globally and then cut down the amount of times you call buildOrganisation(), i.e String organisationName below, would just be organisation.getName()